tests/sentry/api/endpoints/test_organization_incident.py

```diff
@@ -27,12 +27,31 @@ def user(self):
     def test_simple(self):
         self.create_team(organization=self.organization, members=[self.user])
         incident = self.create_incident()
-        other_incident = self.create_incident()
+        other_incident = self.create_incident(status=IncidentStatus.CLOSED.value)
         self.login_as(self.user)
         with self.feature('organizations:incidents'):
             resp = self.get_valid_response(self.organization.slug)
         assert resp.data == serialize([incident, other_incident])
 
+    def test_filter_status(self):
+        self.create_team(organization=self.organization, members=[self.user])
+        incident = self.create_incident()
+        closed_incident = self.create_incident(status=IncidentStatus.CLOSED.value)
+        self.login_as(self.user)
+
+        with self.feature('organizations:incidents'):
+            resp_closed = self.get_valid_response(
+                self.organization.slug, qs_params={
+                    'status': 'closed'})
```

Comment:
You can just pass this as kwargs. `qs_params` is really only needed if you're doing a POST and want to pass querystring params as well.